types/vanillajs-datepicker/DateRangePicker.d.ts

```diff
@@ -0,0 +1,24 @@
+import Datepicker from './Datepicker';
+
+export interface DateRangePickerOptions {
+    inputs?: HTMLElement[];
+    allowOneSidedRange?: boolean;
+}
+export default class DateRangePicker {
+    constructor(element: HTMLElement, options?: DateRangePickerOptions);
+    allowOneSidedRange: boolean;
+    element: HTMLElement;
+    inputs: HTMLElement[];
+    datepickers: Datepicker[];
+
+	get dates(): Date[] | undefined;
```

Comment:
```suggestion
    get dates(): Date[] | undefined;
```
Please use spaces instead of tabs here.